lumigator/infra/mzai/helm/lumigator/Chart.yaml

```diff
@@ -21,10 +21,4 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "0.1.0-alpha.1"
-
-dependencies:
-  - name: postgresql
-    repository: "oci://registry-1.docker.io/bitnamicharts"
-    version: "15.5.20"
-    condition: postgresql.enabled
+appVersion: "1.16.0"
```

Comment:
0.2.0 perhaps?

Reply:
I guess this is the "overall" version, if so shall we set it to 0.1.0-alpha.1 as the backend?